_posts/2019-06-16-good-first-issue.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "goodfirstissue - what, why and how?"
```

Comment:
> "goodfirstissue - what, why and how?"

I don't feel this is descriptive enough, but it could be part of your introductory text.

How about:

> "How I built Good First Issue bot with OpenFaaS Cloud"

or

> Building the Good First Issue bot with OpenFaaS Cloud

Reply:
done